### ISSUE TITLE
Tap Event Fired On Desktop

### ISSUE THREAD

**Issue by bassmanpaul** [ngryman/jquery.finger#38]
Hi,
Quick query; **Is the `tap` event meant to fire the `click` event on desktops as well?** As this throws a spanner in the works for my following scenario:

```js
$( document ).on( 'dblclick', '#btn', function( ev )
{
       // Fire My Logic Off
});

$( document ).on( 'tap', '#btn', function( ev )
{
       $( this ).trigger( 'dblclick' );
});
```
What I expect:

- Desktop - `Single click` event does nothing or other separate event stuff.
- Desktop - `Double click` event to fire the element's logic off.
- Touch phones - `Single tap` to fire the `double click` event and thus fire the element's logic off.

What actually happens:

- Desktop - `Single click` fires the `tap` event which fires my `double click` logic. **Incorrect**
- Desktop - `Double click` too late as `Single click` has fired.
- Touch phones - `Single tap` fires the `double click` and thus fires the element's logic off. **Correct**

So basically my logic fires a click early. This happens in Webkit & FF (not IE11).

http://jsfiddle.net/3cd9wz54/ Shows the `tap` & `click` being fired. 

Could you shed some light on the situation please?

Regards,

Paul
